packages/drivers/odsp-driver/src/contracts.ts

```diff
@@ -205,10 +206,10 @@ export interface IVersionedValueWithEpoch {
     value: any;
     fluidEpoch: string,
     // This is same as "persistedCacheValueVersion" below. This represents the version of data stored in cache.
-    version: 3,
+    version: 4,
 }
 
-export const persistedCacheValueVersion = 3;
+export const persistedCacheValueVersion = 4;
```

Comment:
The whole purpose of organizing it as ISnapshotCachedEntry extends ISnapshotContents was not to bump these versions

Reply:
Yes forgot to revert it back.